config.go

```diff
@@ -34,7 +37,9 @@ func (c *Config) BindArguments(fs *pflag.FlagSet) {
 	fs.StringVarP(&c.username, "username", "u", "", "API user name (required)")
 	fs.StringVarP(&c.password, "password", "p", "", "API user password (required)")
 	fs.BoolVar(&c.insecure, "insecure", false,
-		"Skip verification of the TLS certificates (is needed for the default self signed certificate)")
+		"Skip verification of the TLS certificates (is needed for the default self signed certificate, default false)")
+	fs.BoolVar(&c.doNotUseTLS, "doNotUseTLS", false, "Do NOT use TLS to connect to the gateway (default false)")
```

Comment:
Shouldn't we use POSIX Style here? `--long-option ` not CameCase.

Reply:
indeed

Reply:
:heavy_check_mark: